internal/controller/nnf_node_block_storage_controller.go

```diff
@@ -342,16 +342,14 @@ func (r *NnfNodeBlockStorageReconciler) createBlockDevice(ctx context.Context, n
 			if _, ok := allocationStatus.Accesses[nodeName]; !ok {
 				log.Info("Created storage group", "Id", storageGroupId)
 				allocationStatus.Accesses[nodeName] = nnfv1alpha1.NnfNodeBlockStorageAccessStatus{StorageGroupId: sg.Id}
-
```

Comment:
If we exit the loop here, the nnf_node_block_storage resource prematurely becomes ready.